Bottom: 8d408914962210f68dee36feb1e37047f1f4b585
Top:    e7c72be90e4c7eaa6b18886573d78674a9d0fe04
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2017-01-18 23:51:32 +0000

Don't disable assert when building (clobber -NDEBUG)


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 8babda1..ccbd5e4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -24,10 +24,12 @@ bin_PROGRAMS=	pcb
 
 if DEBUG_BUILD
 # don't disable assert()
+AM_CFLAGS= -Wno-unused-parameter
 else
-AM_CFLAGS= -DNDEBUG
+AM_CFLAGS= -DNDEBUG -Wno-unused-parameter
 endif
 
+
 PCB_SRCS = \
 	action.c \
 	action.h \
